src/object.c

```diff
@@ -185,6 +185,14 @@ robj *createStringObjectFromLongLongForValue(long long value) {
     return createStringObjectFromLongLongWithOptions(value,1);
 }
 
+/* Always creates a string object from a long long, unlike createStringObjectFromLongLong() which
+ * may return an integer object. */
+robj *createStringObjectFromLongLongAlwaysString(long long value) {
```

Comment:
I don't think we should change the existing functionality of `createStringObjectFromLongLong()`. If we're concerned about naming, I think it would make more sense for it to be named `createObjectFromLongLong()` and this function to `createStringObjectFromLongLong()`, but that again will be a rather large change.

An option to avoid adding a new function is to add a `strobj` param to `createStringObjectFromLongLongWithOptions()` that ensures a string is returned, and then call it directly.

Reply:
I would prefer merging it with createStringObjectFromLongLongWithOptions()